crates/red_knot_test/src/parser.rs

```diff
@@ -326,8 +327,25 @@ impl<'s> Parser<'s> {
 
                         self.skip_whitespace();
 
+                        // Parse an optional sequence of `key[=value]` attributes.
+                        let mut keyvals = vec![];
+                        while self.cursor.first().is_alphabetic() {
+                            let Some(key) = self.consume_until(|c| c == '=' || c.is_whitespace())
+                            else {
+                                break;
+                            };
+                            let mut value = None;
+                            if self.cursor.first() == '=' {
+                                value = self.consume_until(char::is_whitespace);
```

Comment:
Will this include a leading `=` in the `value`? Should we use `eat_char` instead of `first` to avoid that?